src/utils/object.js

```diff
@@ -42,10 +42,21 @@ export function buildRecord<Element, Value>(
 /**
  * Properly typed version of `Object.values`.
  */
-export function recordValues<Value>(record: Record<Value>): Value[] {
+export function recordValues<Value>(record: ?Record<Value>): Value[] {
+  if (!record) return [];
   return Object.keys(record).map((key) => record[key]);
 }
 
+/**
+ * Create a shallow copy of given record with removed key.
+ */
+export function recordWithRemovedKey<Value>(record: Record<Value>, key: string): Record<Value> {
```

Comment:
Since it involves an action maybe `removeKeyFromRecord`?

Reply:
I used this during dev, but replaced it with `immer`. Removing that method.